server/graphql/types/src/types/invoice_line.rs

```diff
@@ -206,6 +206,30 @@ impl InvoiceLineNode {
 
         Ok(result.map(InventoryAdjustmentReasonNode::from_domain))
     }
+
+    // todo - from donor_link
+    pub async fn donor_id(&self) -> &Option<String> {
+        &self.row().donor_id
+    }
+
+    // todo - from donor_link
+    pub async fn donor_name(&self, ctx: &Context<'_>) -> Result<Option<String>> {
```

Comment:
Why not just return full NameNode? 👀

Reply:
bc we only need to display the name, might as well not query more than we need? once we do link join, shouldn't need a loader all (example in stocktake PR)

Reply:
Just thinking cause we're already doing loader to query the name row but that's fine